test_data/test_postprocessor/snooty.toml

```diff
@@ -11,3 +11,7 @@ global-write = "Global Writes"
 global-write-clusters = "Global *Clusters*"
 service = "Atlas"
 blank = ""
+
+[deprecated_versions]
+manual = ["v2.2", "v2.4", "v2.6", "v3.0", "v3.2", "v3.4"]
+mms = ["v1.1", "v1.2", "v1.3"]
```

Comment:
Since you're adding this table to the test project, we should be verifying somewhere that it is actually being ingested and saved properly.